src/firebase_functions/private/util.py

```diff
@@ -387,6 +387,8 @@ def timestamp_conversion(time: str) -> _dt.datetime:
     elif precision_timestamp == PrecisionTimestamp.SECONDS:
         return second_timestamp_conversion(time)
 
+    raise ValueError("Invalid timestamp")
```

Comment:
likewise - related to log impl?

Reply:
Again a lint failure here because the function wasn't ending in a return and the if, elif is not exhaustive